charts/illumidesk/Chart.lock

```diff
@@ -0,0 +1,12 @@
+dependencies:
```

Comment:
I would remove this lock file. This is created when you run `helm dependency build` or `update` which will create the charts dependency directory inside your sub chat directory

Reply:
@Abhi94N I left it in the source files since all repos that I have seen include the Chart.lock file to freeze dependencies to specific versions, similar to how a package-lock.json or requirements.txt file work for JavaScript and Python projects, respectively.